src/test/java/edu/cornell/kfs/fp/batch/xml/fixture/DefaultKfsAccountForAwsFixture.java

```diff
@@ -0,0 +1,33 @@
+package edu.cornell.kfs.fp.batch.xml.fixture;
+
+import edu.cornell.kfs.fp.batch.xml.DefaultKfsAccountForAws;
+
+import java.util.Date;
+
+public enum DefaultKfsAccountForAwsFixture {
+    AWS_ABC_KFS_1658328("ABC", "1658328", null),
+    AWS_DEF_KFS_165835X("DEF", "165835X", null),
+    AWS_GHI_KFS_1658498("GHI", "1658498", null),
+    AWS_JKL_KFS_J801000("JKL", "CS*J801000**6600***", null),
+    AWS_MNO_KFS_INTERNAL("MNO", "Internal", null),
+    AWS_PQR_KFS_J801000("PQR", "CS*J801000**6600***", null),
+    AWS_STU_KFS_1023715_INVALID("STU", "IT*1023715*97601*4020*109**AEH56*BAR", null);

```

Comment:
to clean this up, I would add another constructor that just takes in the group and value